test/modules/logicModule/LM_PC_FundingPot_v1.t.sol

```diff
@@ -95,16 +90,1197 @@ contract LM_PC_FundingPot_v1_Test is ModuleTest {
+        vm.warp(roundStart + 1);
+
+        ILM_PC_FundingPot_v1.AccessCriteria memory accessCriteria =
+            _helper_createAccessCriteria(accessCriteriaEnum);
+
+        vm.expectRevert(
+            abi.encodeWithSelector(
+                ILM_PC_FundingPot_v1
+                    .Module__LM_PC_FundingPot__RoundAlreadyStarted
+                    .selector
+            )
+        );
+        fundingPot.editAccessCriteriaForRound(
+            roundId, accessCriteriaId, accessCriteria
+        );
+    }
+
+    function testFuzzEditAccessCriteriaForRound(
+        uint8 accessCriteriaEnumOld,
+        uint8 accessCriteriaEnumNew
+    ) public {
+        vm.assume(accessCriteriaEnumOld >= 1 && accessCriteriaEnumOld <= 4);
+        vm.assume(
+            accessCriteriaEnumNew != accessCriteriaEnumOld
+                && accessCriteriaEnumNew >= 1 && accessCriteriaEnumNew <= 4
+        );
+
+        _helper_setupRoundWithAccessCriteria(accessCriteriaEnumOld);
+        uint64 roundId = fundingPot.getRoundCount();
+        uint8 accessCriteriaId = 0;
+
+        // Create and apply new access criteria
+        ILM_PC_FundingPot_v1.AccessCriteria memory newAccessCriteria =
+            _helper_createAccessCriteria(accessCriteriaEnumNew);
+
+        fundingPot.editAccessCriteriaForRound(
+            roundId, accessCriteriaId, newAccessCriteria
+        );
+
+        // Verify the access criteria was updated
+        (
+            bool isOpen,
+            address nftContract,
+            bytes32 merkleRoot,
+            address[] memory allowedAddresses
+        ) = fundingPot.getRoundAccessCriteria(roundId, accessCriteriaId);
+
+        assertEq(isOpen, accessCriteriaEnumNew == 1);
+        assertEq(nftContract, newAccessCriteria.nftContract);
+        assertEq(merkleRoot, newAccessCriteria.merkleRoot);
+        assertEq(allowedAddresses, newAccessCriteria.allowedAddresses);
+    }
 
     // -------------------------------------------------------------------------
     // Test: Internal Functions
+
+    // -------------------------------------------------------------------------
+    // Helper Functions
+
+    // @notice Creates a default funding round
+    function _helper_createDefaultFundingRound()
+        internal
+        returns (uint, uint, uint, address, bytes memory, bool, bool)
+    {
+        uint roundStart = block.timestamp + 1 days;
+        uint roundEnd = block.timestamp + 2 days;
+        uint roundCap = 1000;
+        address hookContract = address(0);
+        bytes memory hookFunction = bytes("");
+        bool autoClosure = false;
+        bool globalAccumulativeCaps = false;
+
+        return (
+            roundStart,
+            roundEnd,
+            roundCap,
+            hookContract,
+            hookFunction,
+            autoClosure,
+            globalAccumulativeCaps
+        );
+    }
+
+    // @notice calls the create round function
+    function _helper_callCreateRound(
```

Comment:
I still don't understand the case for wrappers like this. If it is just calling the function on the funding pot, why not call it directly and remove that abstraction layer?